### PR TITLE
Added new weather provider: MeteoBlue

### PR DESCRIPTION
**Pull request by gerrieg**
Signed-off-by: Gerhard Riegler <gerhard.riegler@gmail.com>

New Weather provider: https://www.meteoblue.com

### CODE REVIEW

File: bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/model/Wind.java [hunk 1]
```diff
@@ -22,19 +22,25 @@
...
+            @Provider(name = ProviderName.WUNDERGROUND, property = "current_observation.wind_kph"),
             @Provider(name = ProviderName.WUNDERGROUND, property = "avewind.kph"),
             @Provider(name = ProviderName.OPENWEATHERMAP, property = "speed"),
             @Provider(name = ProviderName.FORECASTIO, property = "windSpeed", converter = ConverterType.WIND_MPS),
             @Provider(name = ProviderName.WORLDWEATHERONLINE, property = "windspeedKmph"),
             @Provider(name = ProviderName.YAHOO, property = "wind.speed"),
-            @Provider(name = ProviderName.HAMWEATHER, property = "windSpeedKPH") })
+            @Provider(name = ProviderName.HAMWEATHER, property = "windSpeedKPH"),
+            @Provider(name = ProviderName.METEOBLUE, property = "wind_speed"),
+            @Provider(name = ProviderName.METEOBLUE, property = "wind_speed_max") })
```
Comment (watou) [inline bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/model/Wind.java]: Do `wind_speed` and `wind_speed_max` mean the same thing?  It seems (without research) that they should be two distinct things: the current sample of wind speed, and then a max wind gust speed over some previous period.  For example, "current wind speed 20 km/h, gusting to 45 km/h."

File: bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/model/Wind.java [hunk 2]
```diff
@@ -43,12 +49,15 @@
             @Provider(name = ProviderName.HAMWEATHER, property = "windDirDEG") })
     private Integer degree;
 
-    @ProviderMappings({ @Provider(name = ProviderName.WUNDERGROUND, property = "current_observation.wind_gust_kph"),
+    @ProviderMappings({
+            @Provider(name = ProviderName.WUNDERGROUND, property = "current_observation.wind_gust_kph"),
             @Provider(name = ProviderName.WUNDERGROUND, property = "wind.gust"),
-            @Provider(name = ProviderName.HAMWEATHER, property = "windGustKPH") })
+            @Provider(name = ProviderName.HAMWEATHER, property = "windGustKPH"),
+            @Provider(name = ProviderName.METEOBLUE, property = "wind_gust_max") })
```
Comment (watou) [inline bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/model/Wind.java]: Difference between `wind_speed_max` and `wind_gust_max`?  Just checking. :-)

**Comment by watou** [comment]
Thank you @gerrieg!  Just minor questions, otherwise lgtm.

**Comment by gerrieg** [comment]
```wind_speed``` is for the current condition, ```wind_speed_max``` and ```wind_gust_max``` are only available in the forecast. So the mapping is OK.

**Comment by watou** [comment]
Thanks @gerrieg!

**Comment by watou** [comment]
@gerrieg Could you update the [wiki](https://github.com/openhab/openhab/wiki/Weather-Binding) with the new provider (noting that it is new in 1.9), and also update openhab.cfg and weather.cfg to list the new provider along with the existing ones?  (Sorry I forgot to note the .cfg changes in my review.)

**Comment by gerrieg** [comment]
docs updated and added MeteoBlue to cfg, see https://github.com/openhab/openhab/pull/4581

**Pull request merged**